frontend/templates/product/index.tsx

```diff
@@ -63,6 +64,19 @@ const ProductTemplate: NextPageWithLayout<ProductTemplateProps> = () => {
          category: parseItemcode(selectedVariant.sku ?? '')?.category,
          price: moneyToNumber(selectedVariant.price).toFixed(2),
       });
+      trackGA4ViewItem({
+         currency: selectedVariant.price.currencyCode,
+         value: selectedVariant.price.amount,
+         items: [
+            {
+               item_id: selectedVariant.sku,
+               item_name: selectedVariant.internalDisplayName,
+               item_variant: selectedVariant.id.split('/').pop(),
```

Comment:
What is the id here?

Reply:
we store the variant id as `const variantId = `gid://shopify/ProductVariant/${apiProduct.variant_id}`;` (see `use-cart.ts`) so this then goes and extracts the variant id back out.